dialogflow/stopwatch/Stopwatch/StopwatchService.swift

```diff
@@ -15,14 +15,11 @@
 //
 import Foundation
 import googleapis
-import AuthLibrary
 
 let Host = "dialogflow.googleapis.com"
 let ProjectName = "your-project-identifier" // UPDATE THIS
 let SessionID = "001"
 let SampleRate = 16000
```

Comment:
It'd be good to add comments about what these variables are used for. 

SessionID = "001"  // Session Ids are used in Dialogflow for ...
Same for SampleRate